platform/util/platform-util-unavailableurls/src/test/java/org/codice/ddf/platform/util/http/UnavailableUrlsTest.java

```diff
@@ -121,21 +118,21 @@ public void testUnsuccessfulPing() {
      * @param list - list of two or more positive integers
      * @return true if sequence is exponential, false otherwise
      */
-    private boolean isExponentialBackoff(long[] list) {
+    private boolean isExponentialBackoff(Long[] list) {
```

Comment:
Switched over to object (Long) so I could use a better Hamcrest matcher. The matcher does not accept primitive arrays.